fivetran/rds-writeonly-role.sql

```diff
@@ -0,0 +1,27 @@
+-- creates a write-only role for fivetran to connect to RDS for specific tables
+DO
+$$
+  DECLARE
+    table_name     varchar;
+    project        varchar   := '';
+    environment    varchar   := '';
+    allowed_tables varchar[] := array [
+      'table1',
+      'table2'
+      ];
+    password       varchar   := 'my-super-secret-password';
+  BEGIN
+    EXECUTE format('CREATE ROLE %s_%s_writeonly_fivetran;', project, environment);
```

Comment:
```suggestion
    IF NOT EXISTS (SELECT 1 FROM pg_roles WHERE rolname = format('%s_%s_writeonly_fivetran', project, environment))
THEN
    EXECUTE format('CREATE ROLE %s_%s_writeonly_fivetran;', project, environment);
END IF;
```

this will make the script idempotent. We will need to re-execute it if we add/remove tables from the list.